packages/cmf/src/sagas/http.js

```diff
@@ -286,6 +287,78 @@ export function getDefaultConfig() {
 	return HTTP.defaultConfig;
 }
 
+function* handleHTTPAction({
+	method,
+	type,
+	url,
+	body,
+	transform,
+	onResponse,
+	onError,
+	cmf,
+	...config
+}) {
+	// TODO find a solution to split config, options, cmf related
+	const options = {};
+	let response;
+	if (type === HTTP_METHODS.DELETE) {
+		response = yield call(httpDelete, url, config, options);
+	} else if (type === HTTP_METHODS.GET) {
+		response = yield call(httpGet, url, config, options);
+	} else if (type === HTTP_METHODS.POST) {
+		response = yield call(httpPost, url, body, config, options);
+	} else if (type === HTTP_METHODS.PUT) {
+		response = yield call(httpPut, url, body, config, options);
+	} else if (type === HTTP_METHODS.PATCH) {
+		response = yield call(httpPatch, url, body, config, options);
+	}
+	if (!response.response.ok) {
```

Comment:
I'd like to talk with you about what should be an HTTP error for the HTTP middleware. I'm not sure to agree with the actual implementation, and I'd like to understand why this has been done like that.